ctype.md

```diff
@@ -2,50 +2,48 @@
-    "$id": "KILT_DRIVERS_LICENSE",
+    "$id": "SimpleLicense",
     "$schema": "http://kilt-protocol.org/draft-01/ctype#",
     "properties": {
       "name": {
         "type": "string"
       },
-      "canDrive": {
-        "type": "boolean"
+      "age": {
+        "type": "integer"
       }
     },
     "type": "object"
   },
-  "owner": "8UPfLqrqbKoKyi61YErwUrWEa1PWxikEojV7PCnLJgxrWd6W",
-  "hash": "0x081955a2b7990554f6193a9e770ea625c68d2bfc5a1ff996e6e28d2a620fae16"
+  "owner": "5EkPdXVsoJ8Vm8UTrs7c89ghJpWVZHGN7RGiQFSi6VC6RiX9",
+  "hash": "0x16fa158d730aa83f6339497fafa59d895c89c3eae24dc323532f71e235824152"
 }
 ```
 
-> 💡 JSON Schema  
-> CTYPES are based on JSON Schema, a standard used to annotate and validate JSON documents. We don't need to dive into it in this tutorial, for now we can think of CTYPE as JSON objects.
+> 💡 CTYPES are based on JSON Schema, a standard used to annotate and validate JSON documents. We don't need to dive into it in this tutorial, for now we can think of CTYPE as JSON objects.
 
-Let's have a look at what these attributes are.
+Let's have a look at these attributes.
 
 | Key                     | Value                                                                                                                 |
 | ----------------------- | --------------------------------------------------------------------------------------------------------------------- |
-| `schema` > `$id`        | The name of this CTYPE                                                                                                |
-| `schema` > `properties` | The properties that a claim of type `KILT_DRIVERS_LICENSE` should have                                                |
-| `owner`                 | The public address of the user who created this CTYPE                                                                 |
-| `hash`                  | Most important attribute, the hash is the CTYPE's digital footprint. It's generated from the content of the `schema`. |
+| `schema` > `$id`        | The name of this CTYPE.                                                                                                |
+| `schema` > `properties` | The properties that a claim of type `SimpleLicense` should have.                                                |
+| `owner`                 | The public address of the user who created this CTYPE.                                                                 |
+| `hash`                  | Most important attribute, the hash is the CTYPE's **digital footprint**. |
 
 A CTYPE is stored on the KILT blockchain - more exactly, the CTYPE's hash is stored on the KILT blockchain.
+The full CTYPE can be stored in a regular web service.
 
-In a real-life case:  
-A user would simply retrieve an existing CTYPE from a repository of CTYPES, for example via a REST API.
+In a real-life setup, auser would simply retrieve an existing CTYPE from a repository of CTYPES for example via a REST API.
```

Comment:
Change request: 

add a space to auser

Reply:
Fixed, ty